python-package/xgboost/training.py

```diff
@@ -28,7 +33,7 @@ def _assert_new_callback(
 
 
 def _configure_custom_metric(
-    feval: Optional[Metric], custom_metric: Optional[Metric]
+    feval: FEvalCallable, custom_metric: FEvalCallable
```

Comment:
What's the difference between this and the original `Metric`?

Reply:
It indeed looks unnecessary to introduce a separate `FEvalCallable` as I review the changes I've made. Let me inline it then.